qemu/tests/cfg/virtio_fs_migration_on_error.cfg

```diff
@@ -0,0 +1,88 @@
+- virtio_fs_migration_on_error:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7 Win2012 Win2012..r2 Win8 Win8.1
+
+    type = virtio_fs_migration_on_error
+    virt_test_type = qemu
+    required_qemu = [9.0.0,)
+    kill_vm = yes
+    start_vm = yes
+
+    filesystems = fs
+    filesystems_migration = targetfs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir_fs = /var/tmp/virtio_fs_source
+    fs_source_dir_targetfs = /var/tmp/virtio_fs_target
+    fs_target = myfs
+    fs_dest = /mnt/${fs_target}
+    force_create_fs_source = no
```

Comment:
It might be better if using fs_source_user_config = yes instead.

Reply:
the pre_command and post_command can take care of these well, force_create_fs_source is not used because we also have to handle the target VM's source dir, so let's handle these together

Reply:
> It might be better if using fs_source_user_config = yes instead.

the pre_command and post_command can take care of these well, force_create_fs_source is not used because we also have to handle the target VM's source dir, so let's handle these together

Reply:
Yes, you're right. But what I want to say is if configuring fs_source_user_config = yes , it will not execute the following codes, and then it will save a little time when executing the codes. I know this parameter is useful, but seems that the old codes never consume it, I didn't notice it before.
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/env_process.py#L179

Reply:
Makes sense, if use configuring fs_source_user_config = yes, there are more parameters to configure.
Thanks @zhencliu . This part looks good to me, and for the similar comments in other patches, you can ignore it.

Reply:
Thanks @xiagao for your comments, actually, I almost didn't notice fs_source_user_config at all before you pointed it out, thanks.